app/assets/javascripts/dashboard.js

```diff
@@ -326,7 +319,6 @@ app.controller('PlanController',['$scope','$http', function($scope,$http) {
            }
        }).success(function(data) {
```

Comment:
This should be split up into multiple files to separate your controller logic from declaring your app/module etc.